docs-src/python/backgroundcheck-boilerplate-self-hosted-worker.md

```diff
@@ -0,0 +1,49 @@
+---
+id: backgroundcheck-boilerplate-self-hosted-worker
+title: Customize Client options
+sidebar_label: Self-hosted Client options
+description: Configure the Temporal Client with the specific IP Address of the Temporal Server on your network.
+tags:
+- worker
+- self-hosted
+- developer guide
+---
+
+<!-- DO NOT EDIT THIS FILE DIRECTLY.
+THIS FILE IS GENERATED from https://github.com/temporalio/documentation-samples-python/blob/bgc/backgroundcheck_boilerplate/self_hosted_worker/main_dacx.py. -->
+
+Set IP address, port, and Namespace in the Temporal Client options.
+
+:::copycode Sample application code
+
+The following code sample comes from a working and tested sample application.
+The code sample might be abridged within the guide to highlight key aspects.
+Visit the source repository to [view the source code](https://github.com/temporalio/documentation-samples-python/blob/bgc/backgroundcheck_boilerplate/self_hosted_worker/main_dacx.py) in the context of the rest of the application code.
+
+:::
+
+```python
+import asyncio
```

Comment:
Unused import

Reply:
This is used because the main is being ran.

Reply:
There is no `main` in this snippet

Reply:
There is no code in this snippet using this import though (i.e. there is no `asyncio.run(main())` like there is elsewhere). You don't have to import `asyncio` just to defined `async def main`